ui/src/pages/plan/rules/formUtils.test.ts

```diff
@@ -56,7 +54,7 @@ export function submit(formElement: any) {
 }
 
 export function update(formElement: any) {
-    const updateButton = formElement.getByRole("button", { name: /Update/i });
+    const updateButton = formElement.getAllByRole("button", { name: /Update/i })[0];
```

Comment:
For no good reason, there's a test that renders the form twice? Some sort of bug. Anyway, I'm telling it to click the first update button.